ansible/roles/operator-pipeline/templates/openshift/tasks/upload-artifact.yml

```diff
@@ -37,35 +37,92 @@ spec:
         #! /usr/bin/env bash
         PREFLIGHT_RESULTS_EXISTS="$(params.preflight_results_exists)"
         if [ $PREFLIGHT_RESULTS_EXISTS == "true" ]; then
-          echo "Preflight results already exists- skipping"
+          echo "Preflight logs already exists - skipping"
           echo "none" | tee $(results.log_url.path)
           exit 0
         fi
 
-        echo "Uploading test results and artifacts"
+        echo "Uploading test log"
 
         upload-artifacts \
           --cert-project-id "$(params.cert_project_id)" \
           --operator-version "$(params.bundle_version)" \
           --operator-package-name "$(params.package_name)" \
           --certification-hash "$(params.md5sum)" \
           --pyxis-url "$(params.pyxis_url)" \
-          --artifacts-dir "$(params.artifacts_dir)" \
-          --log-file "$(params.log_file)" \
-          --result-file "$(params.result_file)" \
+          --path "$(params.log_file)" \
+          --type preflight-logs \
+          --output preflight-logs.json \
           --verbose
 
-        cat output.json | jq
+        LOG_ID=$(cat preflight-logs.json | jq -r "._id")
+        LOG_URL="$(params.pyxis_url)v1/projects/certification/artifacts/id/$LOG_ID"
 
-        LOG_ID=$(cat output.json | jq -r ".logs._id")
-        RESULTS_ID=$(cat output.json| jq -r ".test_results._id")
+        echo "Logs URL: "
+        echo -n $LOG_URL | tee $(results.log_url.path)
+
+    - name: upload-test-results
+      image: quay.io/redhat-isv/operator-pipelines-images:latest
+      workingDir: $(workspaces.source.path)
+      env:
+        - name: PYXIS_API_KEY
+          valueFrom:
+            secretKeyRef:
+              name: $(params.pyxis_api_key_secret)
+              key: PYXIS_API_KEY
+      script: |
+        #! /usr/bin/env bash
+        PREFLIGHT_RESULTS_EXISTS="$(params.preflight_results_exists)"
+        if [ $PREFLIGHT_RESULTS_EXISTS == "true" ]; then
+          echo "Preflight results already exists - skipping"
+          echo "none" | tee $(results.result_url.path)
```

Comment:
Should we provide the url to the existing results instead? If not then this special value should be documented in the description of the result.

This applies to the `log_url` result as well.

Reply:
Oh.. so I just found out that getting a logs, results in the hosted pipeline is not yet implemented so we don't have URL yet. I'll add it in AC for getting pipeline artifacts.

Reply:
Ok, thanks.